etcdmain/grpc_proxy.go

```diff
@@ -119,7 +120,6 @@ func startGRPCProxy(cmd *cobra.Command, args []string) {
 	pb.RegisterLeaseServer(server, leasep)
 	pb.RegisterMaintenanceServer(server, mainp)
 	pb.RegisterAuthServer(server, authp)
-	grpc_prometheus.Register(server)
```

Comment:
should there be a `mux.Handle("/metrics", prometheus.Handler())` or something to replace this?